patch/common/cpu.yaml

```diff
@@ -0,0 +1,14 @@
+# Patches for the CPU
+#
+# - Make the prr-register writable
+# - Rename PRTWI and PRSPI fields to match the datasheet
+CPU:
+  _modify:
+    PRR:
+      access: read-write
+  PRR:
+    _modify:
+      PRTWI:
+        name: PRTWI0
+      PRSPI:
+        name: PRSPI0
```

Comment:
Hmm, I am unsure if this is the right thing to do.  Most other chips also have these fields named `PRTWI`/`PRSPI` so we'd have to fix them all.  On the other hand I think the reason why they do not have the suffix number is that the registers for `TWI`/`SPI` also do not have suffix numbers in contrast to e.g. the timer/usart regs.  In general it seems the consensus is to move away from the suffixed register and fields names as this also makes HAL code much easier to write.  So I think I'd keep these field names here as they are as well...

Reply:
If the crate docs don't match the datasheet, then you have to dive into the code to see whether it is a mistake or not. The generated docs don't have the same information as the code, ie what bit positions the field is covering. That's the first thing i had to do, ie check that PRTWI was really PRTWI0 as in the datasheet, I didn't notice it wasn't writable at first. From other work I know there is lots of mistakes in the microchip svd files. OTOH i understand about making HAL more common, except in that case it seems better to have a method on the peripheral to turn off its clock, and just hide the PRR register.

It makes sense to move out of the common file, i can do that.

Thanks for the tip on the compile problem. I'll give that a try when I have time. I'm still a newbie at rust.